### PR TITLE
MON-14220-allow-user-to-use-ip-management-attribute-in-mapper-definit…

### PR DESCRIPTION
**Pull request by YanMeddour**
…ion-with-vm-ware-vm-discovery

## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md [hunk 1]
```diff
@@ -149,6 +149,11 @@ systemctl enable centreon_vmware
 Vous pouvez vérifiez que votre configuration est fonctionelle en consultant les journaux dans :
 "/var/log/centreon/centreon\_vmware.log".
 
+### Balises et Attributs personnalisés
+
+> Pour découvrir les balises et les attributs personnalisés, vous devez utiliser la version 3.2.5 de centreon-vmware-daemon et ajouter --tags dans les options supplémentaires de découverte.                  
+>Allez sur la page Configuration > Hôtes > Découverte, et à la 3ème étape ("Définir les paramètres de découverte"), dans la section Paramètres supplémentaires, dans le champ Options supplémentaires, saisissez --tags.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md:155]:
```suggestion
>Allez à la page **Configuration > Hôtes > Découverte**, et à la 3ème étape (**Définir les paramètres de découverte**), dans la section **Paramètres supplémentaires**, dans le champ **Options supplémentaires**, saisissez **--tags**.
```

File: pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md [hunk 2]
```diff
@@ -151,6 +151,12 @@ systemctl enable centreon_vmware
 Make sure that the daemon configuration works fine by looking for errors in
 "/var/log/centreon/centreon\_vmware.log".
 
+### Tags and Custom Attributes
+
+> To discover Tags and Custom Attributes, you must use version 3.2.5 of centreon-vmware-daemon and add --tags in the discovery extra options.
+
+
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md:158]:
```suggestion
>Go to page **Configuration > Hosts > Discovery**, and at the 3rd step (**Define discovery parameters**), in section **Additional parameters**, in the **Extra Options** field, enter **--tags**.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

File: pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md [hunk 3]
```diff
@@ -151,6 +151,12 @@ systemctl enable centreon_vmware
 Make sure that the daemon configuration works fine by looking for errors in
 "/var/log/centreon/centreon\_vmware.log".
 
+### Tags and Custom Attributes
+
+> To discover Tags and Custom Attributes, you must use version 3.2.5 of centreon-vmware-daemon and add --tags in the discovery extra options.
+
+>Go to page **Configuration > Hosts > Discovery**, and at the 3rd step (**Define discovery parameters**), in section **Additional parameters**, in the **Extra Options** field, enter **--tags**.
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md:158]: Please review this paragraph according to my comment in the french version

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

File: pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md [hunk 4]
```diff
@@ -151,6 +151,13 @@ systemctl enable centreon_vmware
 Make sure that the daemon configuration works fine by looking for errors in
 "/var/log/centreon/centreon\_vmware.log".
 
+### Tags and Custom Attributes
+
+> To discover Tags and Custom Attributes, you must :
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md:156]:
```suggestion
To discover Tags and Custom Attributes, you must : 
```

**Review by omercier — APPROVED** [review]

**Pull request merged**